split-webapp/split/src/components/TransactionList.js

```diff
@@ -19,68 +20,72 @@ class TransactionList extends React.Component {
     super(props);
 
     this.state = {
-        transaction: {
-            users: [],
-            cost: 0,
-            payed: ""
-        }
-    }
+      // If you're using state in component remove the es-lint line below, (else make it a functional component)
```

Comment:
The state here will be used once the api call is completed, for now is just a placeholder. Safe just to leave the eslint ignore for now and keep tracl of it.

Reply:
That sounds good!